docs/source/concepts/DynamicalStructureFactorFromAbInitio.rst

```diff
@@ -129,7 +138,7 @@ where:
 
 with :math:`A`, :math:`B`, :math:`C` as constants.
 
-Moreover, in case of Tosca and Tosca-like instruments length of momentum transfer depends on frequency (*indirect geometry spectrometer*).
+Moreover, in case of TOSCA and TOSCA-like instruments length of momentum transfer depends on frequency (*indirect geometry spectrometer*).
```

Comment:
```suggestion
Moreover, in case of TOSCA and TOSCA-like instruments, the length of momentum transfer depends on frequency (*indirect geometry spectrometer*).
```